### ISSUE TITLE
[Go] Missing template delimiter "}}" breaks highlighting

### ISSUE THREAD

**Issue by radeksimko** [sublimehq/Packages#2402]
- Sublime Version: 3.2.2, Build 3211
- OS Version: macOS 10.15.5

Sometimes invalid template syntax is intentional, but regardless of the reason the interpretation of string should stop at the next quote (`"`) whether the templating syntax inside the string is valid or not.

For example:

```go
func TestGitHubCreateReleaseWrongNameTemplate(t *testing.T) {
	var ctx = context.New(config.Project{
		Release: config.Release{
			NameTemplate: "{{.dddddddddd",
		},
	})
	client, err := NewGitHub(ctx)
	require.NoError(t, err)

	str, err := client.CreateRelease(ctx, "")
	require.Empty(t, str)
	require.EqualError(t, err, `template: tmpl:1: unclosed action`)
}
```

![Screenshot 2020-07-06 at 11 53 21](https://user-images.githubusercontent.com/287584/86586310-992f7f00-bf7f-11ea-8f4d-86233ab875ca.png)

**Comment by jfcherng** [comment]
cannot reproduce with `master` branch.

![image](https://user-images.githubusercontent.com/6594915/86587188-c437c400-bfbb-11ea-9250-97e2e9cbada9.png)

**Comment by deathaxe** [comment]
Fixed by #2086
